website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-okta-deprecated.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Setting up SSO with Okta (Deprecated)"
```

Comment:
Why the 'Deprecated' ?

Reply:
we followed this pattern for GSuite and Azure too. The "deprecated" docs instruct users to send us their SSO creds for setup on our end. Now that this is all self-service, we specifically _do not_ want folks to send their creds our way!

Reply:
and, we intend to promote the native integrations as the recommended/primary way of integrating dbt Cloud with an SSO provider going forwards